### PR TITLE
Direct3D 12: Include in CODEOWNERS

### PR DESCRIPTION
**Pull request by RandomShaper**

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -40,6 +40,7 @@ doc_classes/*                       @godotengine/documentation
 /drivers/gles3/                     @godotengine/rendering
 /drivers/spirv-reflect/             @godotengine/rendering
 /drivers/vulkan/                    @godotengine/rendering
+/drivers/d3d12/                     @godotengine/rendering
```
Comment (YuriSizov) [inline .github/CODEOWNERS:43]: We prefer to stick to the alphabet order :)

Comment (YuriSizov) [reply .github/CODEOWNERS:43]: (Also, don't mind the CI failing for web builds, it's unrelated to your PR).

Comment (RandomShaper) [reply .github/CODEOWNERS]: Fixed. Assuming ASCII order, Direct3D 12 climbs to the first position! 🏆

PS: To add the new entry, I was wondering what the order was. I couldn't realize it was alphabetic. I was thinking, "_Dummy_ comes first as the most basic, OK. Then, OpenGL as the next more sophisticated one. But why is SPIRV-Reflect in the middle?" 😅

Comment (YuriSizov) [reply .github/CODEOWNERS]: Sometimes it just looks so natural, you don't realize it's just the alphabet (and you ignore whatever doesn't fit as a fluke) :P Thanks!

**Review by YuriSizov — APPROVED** [review]

**Comment by YuriSizov** [comment]
Thanks!

**Pull request merged**